src/autoloads/connections.gd

```diff
@@ -0,0 +1,119 @@
+extends Node
+
+enum connectionTypeList {
```

Comment:
perhaps make this variable just be `connectionType` or `connectionTypes` for more readability when accessing these types

Reply:
changed